app/redhat_insights/default/data/ui/views/main_dashboard.xml

```diff
@@ -642,7 +642,7 @@
 					"earliest": "-24h@h",
 					"latest": "now"
 				},
-				"query": "index=redhatinsights AND bundle = \"rhel\" AND event_type=\"compliance-below-threshold\" AND earliest=-24h@m AND latest=+30d@m\n| spath \n| rename object.* as *\n| spath path=events{} output=events\n| mvexpand events\n| spath path=context{} output=context\n| mvexpand context\n| eval _raw=events\n| kv\n| eval name=case(event_type=\"compliance-below-threshold\", 'payload.policy_name')\n| eval threshold=case(event_type=\"compliance-below-threshold\", 'payload.policy_threshold')\n| eval version=mvindex('context.rhel_version',0)\n| rex field=version \"(?<majorVersion>\\w+)\\.(?<minorVersion>\\w+)\"\n| strcat \"RHEL \" majorVersion finalVersion\n| stats count(\"payload.policy_name\") as events_count by \"payload.policy_id\", name, threshold, finalVersion\n| appendpipe [| stats count | where count=0 | addinfo \n| eval name=\"---\"  \n| eval finalVersion=\"---\"\n| eval events_count=\"---\"\n| eval threshold=\"---\"\n]\n| table name, finalVersion, events_count, threshold\n| sort -events_count\n| rename name as \"Policy name\", finalVersion as \"RHEL Version\", events_count as \"Events\", threshold as \"Threshold\"\n| head 3"
+				"query": "index=redhatinsights AND bundle = \"rhel\" AND event_type=\"compliance-below-threshold\" AND earliest=-30d@d AND latest=now\n| spath \n| rename object.* as *\n| spath path=events{} output=events\n| mvexpand events\n| spath path=context{} output=context\n| mvexpand context\n| eval _raw=events\n| kv\n| eval name=case(event_type=\"compliance-below-threshold\", 'payload.policy_name')\n| eval threshold=case(event_type=\"compliance-below-threshold\", 'payload.policy_threshold')\n| eval version=mvindex('context.rhel_version',0)\n| rex field=version \"(?<majorVersion>\\w+)\\.(?<minorVersion>\\w+)\"\n| strcat \"RHEL \" majorVersion finalVersion\n| stats count(\"payload.policy_name\") as events_count by \"payload.policy_id\", name, threshold, finalVersion\n| appendpipe [| stats count | where count=0 | addinfo \n| eval name=\"---\"  \n| eval finalVersion=\"---\"\n| eval events_count=\"---\"\n| eval threshold=\"---\"\n]\n| table name, finalVersion, events_count, threshold\n| sort -events_count\n| rename name as \"Policy name\", finalVersion as \"RHEL Version\", events_count as \"Events\", threshold as \"Threshold\"\n| head 3"
```

Comment:
Please consolidate the first query search with `queryParameters` for this whole query.
* they do not match
* and, we should use a single definition, preferably with `queryParameters`

Reply:
There are query params for the query:

```
"queryParameters": {
    "earliest": "-24h@h",
    "latest": "now"
},
```

vs

```
index=redhatinsights ... AND earliest=-30d@d AND latest=now ...
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^          
```


```
		"ds_btBzUHEq": {
			"type": "ds.search",
			"options": {
				"queryParameters": {
					"earliest": "-24h@h",
					"latest": "now"
				},
				"query": "index=redhatinsights AND bundle = \"rhel\" AND event_type=\"compliance-below-threshold\" AND earliest=-30d@d AND latest=now\n| spath \n| rename object.* as *\n| spath path=events{} output=events\n| mvexpand events\n| spath path=context{} output=context\n| mvexpand context\n| eval _raw=events\n| kv\n| eval name=case(event_type=\"compliance-below-threshold\", 'payload.policy_name')\n| eval threshold=case(event_type=\"compliance-below-threshold\", 'payload.policy_threshold')\n| eval version=mvindex('context.rhel_version',0)\n| rex field=version \"(?<majorVersion>\\w+)\\.(?<minorVersion>\\w+)\"\n| strcat \"RHEL \" majorVersion finalVersion\n| stats count(\"payload.policy_name\") as events_count by \"payload.policy_id\", name, threshold, finalVersion\n| appendpipe [| stats count | where count=0 | addinfo \n| eval name=\"---\"  \n| eval finalVersion=\"---\"\n| eval events_count=\"---\"\n| eval threshold=\"---\"\n]\n| table name, finalVersion, events_count, threshold\n| sort -events_count\n| rename name as \"Policy name\", finalVersion as \"RHEL Version\", events_count as \"Events\", threshold as \"Threshold\"\n| head 3"
			},
			"name": "search_compliance_table"
```